lib/dealership.rb

```diff
@@ -0,0 +1,14 @@
+class Dealership
+  attr_reader :dealership, :address, :inventory,
+  :inventory_count
+  def initialize(dealership, address)
+    @dealership = dealership
+    @address = address
+    @inventory = []
+    @inventory_count = 0
+  end
+
+  def add_car
+    @inventory << @car
```

Comment:
`@car` is not a class-level variable, so should not use the `@`. Instead, this method should take an argument of `car`, since in your tests you are passing cars to this method.